src/Controls/tests/TestCases.Shared.Tests/UITestIgnoreAttributes.cs

```diff
@@ -0,0 +1,102 @@
+﻿using NUnit.Framework;
+
+namespace Microsoft.Maui.TestCases.Tests
+{
+	public class FailsOnAllPlatforms : IgnoreAttribute
+	{
+		public FailsOnAllPlatforms() : base(nameof(FailsOnAndroid))
+		{
+		}
+		public FailsOnAllPlatforms(string reason) : base(reason)
+		{
+		}
+	}
+
+#if ANDROID
+	public class FailsOnAndroid : IgnoreAttribute
```

Comment:
Don't we have this in a different place too? Can we move it to somewhere so it's shared maybe?